packages/core/src/redux/slices/rtcConnection/index.ts

```diff
@@ -215,7 +215,7 @@ export const doHandleAcceptStreams = createAppThunk((payload: StreamStatusUpdate
     dispatch(isAcceptingStreams(true));
     const state = getState();
     const rtcManager = selectRtcConnectionRaw(state).rtcManager;
-    const remoteParticipants = selectRemoteParticipants(state);
+    const remoteParticipants = selectRemoteClients(state);
```

Comment:
This local var should be renamed also I think

Reply:
fixed